### ISSUE TITLE
Electric cart enhancement

### ISSUE THREAD

**Issue by ghost** [PilzAdam/carts#1]
As i wrote on the forum, an idea would be an electric powered cart, where W speeds up and S speeds down, made with (C -> cart; B -> batteries):

B B B
B C B
B B B

Eight batteries, the cart would have the charge of battery*8, and the electric cart block may be charged on the battery box.
Mesecon energy could load batteries.

Power drain could depend on speed.

More power is needed? Battery box cart would be good, as made from cart mk1 (the one made by batteries) and battery boxes instead of batteries on recipe, total power given by the sum of the mk1 cart power plus eight battery box. And for cart mk3, MV battery box with the same as mentioned before.

In short:

Electric cart MK1 (Battery - Cart)

B   B   B
B   C   B
B   B   B

Electric cart MK2 (Battery Box - Cart Mk1)

BB   BB   BB
BB   C1   BB
BB   BB   BB

Electric cart MK3 (MV Battery Box - Cart Mk2)

MVBB  MVBB  MVBB
MVBB     C2    MVBB
MVBB  MVBB  MVBB

**Comment by Uberi** [comment]
Perhaps this is better suited as part of Technic.

**Comment by jojoa1997** [comment]
No don't make people download something as big as technic. I say add powered carts, and chest carts
